src/main.cpp

```diff
@@ -14,6 +14,8 @@
 #include "sensors/d200.hpp"
 #include "sensors/ACS712.hpp"
 
+#include <TeensyDebug.h>
```

Comment:
stray include in main, I don't see it being used here.

Reply:
The "debug" object comes from this